images/opensuse/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# Default to tumbleweed distro
+ARG RELEASE=tumbleweed
```

Comment:
No need to default as this ARG should always be required/specified during a build.